code/modules/projectiles/ammunition/ballistic/pistol.dm

```diff
@@ -33,8 +33,13 @@
 	name = "9mm armor-piercing bullet casing"
 	desc = "A 9mm armor-piercing bullet casing."
 	projectile_type =/obj/projectile/bullet/c9mm_ap
+	
+/obj/item/ammo_casing/c9mm/hp
+	name = "9mm hollow-point bullet casing"
+	desc = "A 10mm hollow-point bullet casing."
+	projectile_type = /obj/projectile/bullet/c9mm_hp
 
-/obj/item/ammo_casing/c9mm/inc
+/obj/item/ammo_casing/c9mm/fire

```

Comment:
`/obj/item/ammo_casing/c9mm/incendiary` might be better
`fire` is used for a lot of things on the codebase

Reply:
It's not really standardized. I could go through and do all that just to tidy it up and make it more readable

Reply:
That'd be a pretty positive change, though it's not really required.